### ISSUE TITLE
Suggestion: The product list 

### ISSUE THREAD

**Issue by Eduardo-Vieira** [keikaavousi/fake-store-api#99]
Suggestion: The product list should have pagination, and add more items

[mongoose-paginate](https://www.npmjs.com/package/mongoose-paginate-v2)

**Comment by Sun3** [comment]
I agree, you need a lot more products to display. Adding paginations would bring it to a higher standard.

Thanks.

**Comment by Bilalzip** [comment]
Suggestion: I need products to be sorted out by color , theme please add these details in schema also

**Comment by keikaavousi** [comment]
Thank you for your suggestion!
Sure! I will include these items in the next version.
Happy coding!
